.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+charset = utf-8
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[{package*.json,lerna.json}]
+indent_size = 2
```

Comment:
Why multiple indent sizes? I generally prefer 2, but am mainly just curious about this 4/2 special case.

Reply:
NPM defaults to using 2 spaces, so this ignores it to keep it consistent

Reply:
I looked and I think most of the other ARIA specs use 2 (except where there are errors, obvs)

- aria/index.html
- core-aam/index.html
- graphics-aam/index.html
- others

The CSS and JS files are less consistent, by on quick glance, more (?) of the JS files use 2 spaces as well, even several of the JS files in this diff.

I'll defer to the Chairs and editors (@spectranaut, @pkra, @jnurthen) but my preference would be to change as little as possible in the source, and therefore keep the default spacing to 2 spaces per indent.